source/libs/dom-utils.ts

```diff
@@ -38,7 +38,7 @@ export const safeElementReady = selector => {
  *   <nope/>
  * </parent>
  */
-export const appendBefore = (parent, before, child) => {
+export const appendBefore = (parent: string|Element, before: string|Element, child: Element) => {
```

Comment:
If you change the type of `before` to be an union type, you should update as well the comment above the function signature.